src/codeu/chat/client/commandline/Chat.java

```diff
@@ -491,6 +498,111 @@ private ConversationHeader findConversation(String title) {
     return null;
   }
 
+  private Panel createPlayPanel(final UserContext user) {
+
+    final Panel panel = new Panel();
+
+    panel.register("help", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        System.out.println("  titles");
+        System.out.println("    Display titles of all available plays.");
+        System.out.println("  all");
+        System.out.println("    Display all ongoing plays and their statuses.");
+        System.out.println("  new <title>");
+        System.out.println("    Create and join a new play enactment for the given title.");
+        System.out.println("  join <title>");
+        System.out.println("    Join an ongoing play enactment for the given title.");
+        System.out.println("  back");
+        System.out.println("    Go back to the user panel.");
+      }
+    });
+
+    panel.register("titles", new Panel.Command() {
```

Comment:
General advice: Put comments on top describing what each of these commands does. Imagine coming back to edit this code a year later; won't you have a much easier time understanding what's happening if you had some helpful comments? :)